core/src/main/scala/design/hamu/core/command/Command.scala

```diff
@@ -8,5 +8,6 @@ object Command {
 
   def apply(string: String): Command = string match {
     case "greet" => GreetCommand
+	case "fib" => FibCommand
```

Comment:
CI pipeline is yelling about code formatting. You can fix by running `scalafmtAll` in sbt. :+1: